src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/InheritableTaglet.java

```diff
@@ -52,7 +52,7 @@ public interface InheritableTaglet extends Taglet {
      * In the future, this could be reworked using some other mechanism,
      * such as throwing an exception.
```

Comment:
Should `ThrowsTaglet.Failure` (eventually?) be declared here in `InheritableTaglet`